spec/javascripts/spec_helper.js

```diff
@@ -0,0 +1,11 @@
+//= require support/json2
```

Comment:
In Sprockets 4 the top level/wrapper asset sets the MIME type that all its included assets will have. If we use ```spec_helper.coffee``` any attempt to include a JS file (such as ```support/json2```) will fail since Sprockets can not convert JS to CoffeeScript. [See this comment] (https://github.com/rails/sprockets/issues/273#issuecomment-206415863) for a slightly longer explanation.